jobs/pr_gate/docker/monorail/config.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
FYI, the configuration used when running tests via run_tests.py is retrieved from test/config/rackhd_defaults.json.   This docker setup will deploy config.json from here.
So using this config to set the system up, and then using that test/config/rackhd_default.json to actually run the tests may conflict.
I suggest we copy this file to test/config/rackhd_default.json as soon as we switch to using docker.
then we can look at updating run_tests.py to grab config.json from the deployed system instead of using it from the config directory.  (they may even match now) :-)